homeassistant/components/shelly/strings.json

```diff
@@ -138,6 +138,24 @@
       }
     },
     "sensor": {
+      "gas_detected": {
+        "state": {
+          "none": "None",
+          "mild": "Mild",
+          "heavy": "Heavy",
+          "test": "Test"
+        },
+        "state_attributes": {
+          "options": {
+            "state": {
+              "none": "None",
+              "mild": "Mild",
+              "heavy": "Heavy",
+              "test": "Test"
+            }
+          }
+        }
```

Comment:
If I understand correctly, these are not related to this PR but to the `extra_state_attributes`?

Reply:
Nope, @bieniu pointed me out that also options needs to be translated so I added those.

Reply:
I think what he pointed was to the `extra_state_attributes` of an entity, this entity has only states, but maybe I am wrong. I'll let him review the translations

Reply:
https://github.com/home-assistant/core/pull/140386#discussion_r1998113694

Reply:
I will let him answer, but anyhow if they point to the same translation we should refer to the existing one using its key - see https://github.com/home-assistant/core/pull/140662 for example

Reply:
Sorry, I missed a notification, as Shay mentioned, references to the existing keys should be used here.

Reply:
I already pushed the related change